### ISSUE TITLE
Print gathered results after interrupted run

### ISSUE THREAD

**Issue by mvz** [backus/mutest#34]
mutest's run time may be very long if there are many mutations to consider. If the user interrupts this, it would be useful to print the set of live mutations already discovered (similar to how RSpec prints failed specs).

**Comment by dgollahon** [comment]
@mvz: Yeah, I agree. I've been wanting that myself for a while. There's definitely work to be done on the reporter.
